easybuild/tools/robot.py

```diff
@@ -154,7 +155,7 @@ def resolve_dependencies(unprocessed, build_specs=None, retain_all_deps=False):
         if loopcnt > maxloopcnt:
             tup = (maxloopcnt, unprocessed, irresolvable)
             msg = "Maximum loop cnt %s reached, so quitting (unprocessed: %s, irresolvable: %s)" % tup
-            _log.error(msg)
+            raise EasyBuildError(msg)
```

Comment:
it would be really good if we could do `EasyBuildError(template, *args)`. lots of these temp variables can be cleaned up then